src/libmongoc/doc/installing.rst

```diff
@@ -141,162 +186,133 @@ Additionally, it is possible to build only libbson by setting the ``-DENABLE_MON
 
 A build configuration description similar to the one above will be displayed, though with fewer entries. Once the configuration is complete, the selected items can be built and installed with these commands:
 
-.. code-block:: none
-
-  $ sudo make install
-
-There are two ways to uninstall the components that have been installed.  The first is to invoke the uninstall program directly.  On Linux/Unix:
-
-.. code-block:: none
-
-  $ sudo /usr/local/share/mongo-c-driver/uninstall.sh
-
-On Windows:
-
-.. code-block:: none
-
-  $ C:\mongo-c-driver\share\mongo-c-driver\uninstall.bat
-
-The second way to uninstall is from within the build directory, assuming that it is in the exact same state as when the install command was invoked:
+Preparing a build from a git repository clone
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. code-block:: none
-
-  $ sudo make uninstall
-
-The second approach simply invokes the uninstall program referenced in the first approach.
+Clone the repository and prepare the build on the current master branch or a particular release tag:
 
-Building from git
-^^^^^^^^^^^^^^^^^
-
-Clone the repository and build the current master or a particular release tag:
-
-.. code-block:: none
+.. parsed-literal::
 
   $ git clone https://github.com/mongodb/mongo-c-driver.git
   $ cd mongo-c-driver
-  $ git checkout x.y.z  # To build a particular release
+  $ git checkout |version|  # To build a particular release
   $ python build/calc_release_version.py > VERSION_CURRENT
   $ mkdir cmake-build
   $ cd cmake-build
   $ cmake -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF ..
-  $ sudo make install
 
-Generating the documentation
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Preparing a build on Windows with Visual Studio
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Install `Sphinx <http://www.sphinx-doc.org/>`_, then:
+On the Windows platform with Visual Studio, it may be necessary to specify the CMake generator to use.  This is especially important if multiple versions of Visual Studio are installed on the system or if alternate built tools (e.g., MinGW, MSYS2, Cygwin, etc.) are present on the system.  Specifying the generator will ensure that the build configuration is known with certainty, rather than relying on the toolchain that CMake happens to find.
 
-.. code-block:: none
+Let's start by generating Visual Studio project files. The following assumes we are compiling for 64-bit Windows using Visual Studio 2015 Express, which can be freely downloaded from Microsoft. We will be utilizing ``cmake``'s out-of-source build feature to keep build artifacts separate from source files.
```

Comment:
Remove "Let's" for consistency with other instructions. Most of the language in this guide is imperative, and uses "you" instead of "we". I suggest changing "assumes we" to "assumes you" and "We will be utilizing" to "Utilize" (out-of-source builds seems like the general recommendation, so making it imperative seems ok).

Reply:
Done.